back/app/routes.py

```diff
@@ -1,5 +1,17 @@
-from app import app
+from app import app,db 
+from flask import jsonify, request
 
 @app.route('/')
 def home():
     return "Hello, Flask!"
+
+@app.route('/get_all', methods=['GET'])
+def get_all():
+    try:
+        if request.method == 'GET':
+         condition = {"status": {"$ne": "closed"}}
```

Comment:
AI?
the status has only 2 values "open" and **`"close"`** 
change "closed" -> "close"